File: src/m/materials/TMeltingPoint.py

def TMeltingPoint(reftemp, pressure):
    '''
    Calculate the pressure melting point of ice at a given pressure

    reftemp is the melting temperature in K at atmospheric pressure (initialized in md.materials.meltingpoint)

    pressure is in Pa

    Usage:
        Tm = TMeltingPoint(md.materials.meltingpoint, pressure)
    '''

    #variables
    beta = 7.9e-8

    #ensure ref is same dimension as pressure
    ref = reftemp - beta * pressure

    return ref

File: src/m/materials/TMeltingPoint.m
function Tm=TMeltingPoint(reftemp, pressure)
%TMELTINGPOINT- calculate pressure melting point of ice
%
%   reftemp is the melting temperature at atmospheric pressure (initialized in md.materials.meltingpoint)   
%
%   pressure is in Pa   
%
%   Usage:
%   Tm=TMeltingPoint(md.materials.meltingpoint,pressure)

%variables
beta=7.9e-8; % K Pa^-1

Tm=reftemp-beta*pressure;

